ui/perfherder/TableAverage.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SimpleTooltip from '../shared/SimpleTooltip';
+
+import TooltipGraph from './TooltipGraph';
+import { displayNumber } from './helpers';
+
+const TableAverage = ({ value, stddev, stddevpct, replicates }) => {
+  let tooltipText;
+  if (replicates.length > 1) {
+    tooltipText = `Runs: < ${replicates.join(' ')} > ${displayNumber(
+      stddev,
+    )} = ${displayNumber(stddevpct)}% standard deviation)`;
+  } else if (replicates.length === 1) {
+    tooltipText = 'Only one run (consider more for greater confidence)';
+  }
+
+  const notZeroSum =
+    replicates.length > 1
+      ? replicates.reduce((accumulator, current) => accumulator + current)
+      : false;
```

Comment:
Could simplify this slightly to:

`const notZeroSum = replicates.length && replicates.reduce((accumulator, current) => accumulator + current)`